tests/test_index.py

```diff
@@ -606,8 +606,6 @@ def _build_test_index(workdir):
     """
     Copy repodata.json, packages to workdir for testing.
     """
-
-    # Python 3.7 workaround "no dirs_exist_ok flag"
```

Comment:
`shutil.copytree()` gained a `dirs_exist_ok` flag in 3.8 https://docs.python.org/3/library/shutil.html#shutil.copytree. If you don't pass that flag, then `shutil.copytree(folder, existing_target_folder)` fails. So this code copies files inside existing-target-folder instead of on-top-of.

https://github.com/conda/conda-index/blob/90879f56e23b1d7b1c14fa632f5d20163130dfc8/tests/test_index.py#L422 already replaced conda-build's version.

Reply:
Thanks for the insights!